bazel/repositories.bzl

```diff
@@ -235,6 +235,7 @@ def _pl_cc_toolchain_deps():
     cc_toolchain_config_repo("unix_cc_toolchain_config", patch = "//bazel/cc_toolchains:unix_cc_toolchain_config.patch")
 
 def _pl_deps():
+    _bazel_repo("bazel_skylib")
```

Comment:
A little odd listing this twice, but I don't really have a cleaner suggestion without creating a multi stage thing like tensorflow.

Reply:
Yeah, both `rules_py` and our `cc_toolchains` depend on it and so we need this to be setup before either. I could pull it into a func of it's own but that seems strange too.